File: 27-RemoveElement/27-RemoveElement.py
# Last updated: 11/11/2025, 1:58:05 AM
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """

        k = 0

        for i in range(len(nums)):
            if nums[i] != val:
                nums[k]=nums[i]
                k += 1

        return k


        
            

File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 11/11/2025, 1:58:08 AM
class Solution {
    public int removeElement(int[] nums, int val) {
        int counter = 0;
        for (int i = 0; i<nums.length; i++) {
            if (nums[i] != val) {
                nums[counter] = nums[i];
                counter++;
                }
            } 
            return counter;
        }
    }